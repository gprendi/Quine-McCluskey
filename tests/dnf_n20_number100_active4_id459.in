d 20
00100000000001000100
00100010000000000100
00000000000001000000
00000000000000000000
00000000000000000100
00100010000001000100
00100000000000000000
00100000000000000100
00000010000000000000
00000010000001000100
00000010000001000000
00000010000000000100
00000000000001000100
00100010000001000000
00100010000000000000
00100000000001000000
