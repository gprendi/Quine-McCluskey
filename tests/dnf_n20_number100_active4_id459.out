p dnf 20 0
-1 -2 -4 -5 -6 -8 -9 -10 -11 -12 -13 -15 -16 -17 -19 -20 0
